temporal-serviceclient/src/main/java/io/temporal/serviceclient/RpcRetryOptions.java

```diff
@@ -323,6 +348,8 @@ public RpcRetryOptions validateBuildWithDefaults() {
 
   private final Duration initialInterval;
 
+  private final Duration longInitialInterval;
```

Comment:
"Long" is terribly unhelpful in understanding the purpose. It's the quality of the value, not a description of the purpose. Let's rename to something like "congestionInitialInterval"